lib/ansible/runner.rb

```diff
@@ -217,14 +223,19 @@ def async?(ansible_runner_method)
         ansible_runner_method == "start"
       end
 
-      def runner_params(base_dir, ansible_runner_method, playbook_or_role_args)
+      def runner_params(base_dir, ansible_runner_method, playbook_or_role_args, verbosity)
         runner_args = playbook_or_role_args.dup
 
         runner_args.delete(:roles_path) if runner_args[:roles_path].nil?
 
         runner_args[:role_skip_facts] = nil if runner_args.delete(:role_skip_facts)
         runner_args[:ident] = "result"
 
+        if verbosity > 0
```

Comment:
is `nil` a possibility here for `verbosity`?  Do we want to have a `.to_i` guard here for safety?

Reply:
I don't think so. I default the arg to `0` everywhere there is a default value and do the `to_i` here https://github.com/ManageIQ/manageiq/pull/18989/files#diff-96e20f5faa69bc61958955255cb5341fR39.

I can also add it to the `validate_params!` method if you still feel like there's a chance it could come in as something other than an integer.

Reply:
Yeah, my thought was that if this method was ever called through other means, those don't matter.  Figured that a `.to_i` is a no-op for integers, and defaults to `0` with `nil`, so it should really shouldn't change what you have already.

Up to you.